src/amdgpu_libdrm.cpp

```diff
@@ -51,6 +52,13 @@ static int libdrm_initialize() {
         return -1;
     }
 
+    char *renderD = drmGetRenderDeviceNameFromFd(fd);
+    fd = open(renderD, O_RDWR);
```

Comment:
Err, please use the DRMDevice API, don't open one node only to get another.

Reply:
Sorry, I've never really worked with libdrm before. Will fix shortly